queries.sql

```diff
@@ -7,3 +7,27 @@ SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
 
+START TRANSACTION;
+UPDATE animals SET species='unspecified';
+ROLLBACK;
```

Comment:
- [ ] @shaaibu7 great work on your query however you need to add queries before and after the rollback to verify changes were made ad then unmade after the rollback.